apps/landingPage/containers/DeveloperExperience.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+
+import Content from '../components/Content';
+import Highlight from '../components/Highlight';
+
+export default function DeveloperInfo() {
+  return (
+    <Content id="developer-experience">
+      <Items>
+        <Item>
+          <ImageContainer>
+            <Image
+              src="/static/icon-multi-platform.jpg"
+              srcSet={`/static/icon-multi-platform.jpg 1x,
+                /static/icon-multi-platform@2x.jpg 2x, /static/icon-multi-platform@3x.jpg 3x`}
+              alt="Multiplatform app"
+            />
+          </ImageContainer>
+          <Text>Truly </Text>
+          <HighlightWrapper>
+            <Highlight>multi-platform app </Highlight>
+          </HighlightWrapper>
+          <Description>
+            One codebase for iOS, Android and web. And with user experience like
+            in native apps.
+          </Description>
+        </Item>
+        <Item>
+          <ImageContainer>
+            <Image
+              src="/static/icon-modern-technologies.jpg"
+              srcSet={`/static/icon-modern-technologies.jpg 1x,
+                /static/icon-modern-technologies@2x.jpg 2x, /static/icon-modern-technologies@3x.jpg 3x`}
+              alt="React logo"
+            />
+          </ImageContainer>
+          <Text>Developed with </Text>
+          <HighlightWrapper>
```

Comment:
What do you think of extending style of Highlight
`const HighlightLeftOffset = styled(Highlight)`
  position: relative;
  left: -5px;
`;` instead of wrapping component?

Reply:
Cool! I didn't know about that

Reply:
Done